polaris-react/src/components/PortalsManager/components/PortalsContainer/PortalsContainer.tsx

```diff
@@ -1,12 +1,16 @@
 import React, {forwardRef, RefObject} from 'react';
 
+import {useUniqueId} from '../../../../utilities/unique-id';
+
 export interface PolarisContainerProps {}
 
 function PortalsContainerComponent(
   _props: PolarisContainerProps,
   ref: RefObject<HTMLDivElement>,
 ) {
-  return <div id="PolarisPortalsContainer" ref={ref} />;
+  const uniqueId = useUniqueId('PortalsContainer');
```

Comment:
(Fixes duplicate ID pa11y CI errors)

Reply:
This should be resolved with a merge from `main`. This happened when two `AppProviders` wrapped a component as a bug caused by me 😭

Reply:
Ahh okay hank you!! This CI check was driving me nuts 😁

Reply:
My bad Chloe.

Reply:
No worries!!!

Reply:
It looks good now. When there are zero changes in `polaris-react` the a11y test doesn't run.

Reply:
https://github.com/Shopify/polaris/blob/fdf4d836dccedd4578d094fa216c5460780064cc/.github/workflows/ci-a11y-vrt.yml#L3-L16

Reply:
Ok I am silly. Fixed here 31b860374fc101c55792f7dbfb0ffc4f67179ed1

Reply:
Sorry for the inconvenience Chloe 😬